apex/contrib/optimizers/distributed_fused_adam.py

```diff
@@ -1032,6 +1052,39 @@ def parameters(self) -> Iterable[torch.nn.Parameter]:
             group["params"] for group in self.param_groups
         )
 
+    def parameter(
+        self,
+        *args: Union[int, ParameterFragment],
+    ) -> torch.nn.Parameter:
+        """Get optimizer parameter
+
+        Can either accept two ints or one
+        DistributedFusedAdam.ParameterFragment.
+
+        Arguments:
+            param_group_id (int): Parameter group index
+            param_id (int): Parameter index within parameter group
+
+        """
+        if (
+            len(args) == 2
+            and isinstance(args[0], int)
+            and isinstance(args[1], int)
+        ):
+            param_group_id = args[0]
+            param_id = args[1]
+        elif len(args) == 1 and isinstance(args[0], self.ParameterFragment):
+            fragment = args[0]
+            param_group_id = fragment.param_group_id
+            param_id = fragment.param_id
+        else:
+            raise TypeError(
+                "Expected input types are "
+                "[int, int] or [DistributedFusedAdam.ParameterFragment], "
+                "but found {[type(arg).__name__ for arg in args]}"
```

Comment:
I'm not sure but would this happen to miss `f`?
```suggestion
                f"but found {[type(arg).__name__ for arg in args]}"
```